compiler/arm/codegen/TreeEvaluatorTable.hpp

```diff
@@ -38,13 +38,27 @@
    TR::TreeEvaluator::bloadEvaluator,       // TR::bload
    TR::TreeEvaluator::sloadEvaluator,       // TR::sload
    TR::TreeEvaluator::lloadEvaluator,       // TR::lload
+   TR::TreeEvaluator::iloadEvaluator,       // TR::irdbar
+   TR::TreeEvaluator::floadEvaluator,       // TR::frdbar
+   TR::TreeEvaluator::dloadEvaluator,       // TR::drdbar
+   TR::TreeEvaluator::aloadEvaluator,       // TR::ardbar
+   TR::TreeEvaluator::bloadEvaluator,       // TR::brdbar
+   TR::TreeEvaluator::sloadEvaluator,       // TR::srdbar
+   TR::TreeEvaluator::lloadEvaluator,       // TR::lrdbar
    TR::TreeEvaluator::iloadEvaluator,       // TR::iloadi
    TR::TreeEvaluator::floadEvaluator,       // TR::floadi
    TR::TreeEvaluator::dloadEvaluator,       // TR::dloadi
    TR::TreeEvaluator::aloadEvaluator,       // TR::aloadi
    TR::TreeEvaluator::bloadEvaluator,       // TR::bloadi
    TR::TreeEvaluator::sloadEvaluator,       // TR::sloadi
    TR::TreeEvaluator::lloadEvaluator,       // TR::lloadi
+   TR::TreeEvaluator::iloadEvaluator,       // TR::irdbar
+   TR::TreeEvaluator::floadEvaluator,       // TR::frdbar
+   TR::TreeEvaluator::dloadEvaluator,       // TR::drdbar
+   TR::TreeEvaluator::aloadEvaluator,       // TR::ardbar
+   TR::TreeEvaluator::bloadEvaluator,       // TR::brdbar
+   TR::TreeEvaluator::sloadEvaluator,       // TR::srdbar
+   TR::TreeEvaluator::lloadEvaluator,       // TR::lrdbar
```

Comment:
Should all these comments have an "i" at the end (i.e. `TR::irdbar` &rarr; `TR::irdbari`)?